Automattic/simplenote-macos
# Updates TableView Styles

### Fix
In this PR we're enhancing **TableRowView** so that it renders an inset squared corner highlight, when running in Big Sur.

@eshurakov May I trouble you yet again?

**Ideally** it'd be neat to verify this in Big Sur / Catalina (we'd need to set you up with Parallels). But if time is a constraint (at least this week), verifying regressions in Catalina would be super neat.

cc @SylvesterWilmott 

Thanks in advance!!
Ref. #718

---

### Test: Tags List
1. Log into the app

- [ ] Verify the first row's vertical alignment (Tags List) matches the first row in the Notes List
- [ ] Verify the selected Tag is highlighted with a (full width) background color
- [ ] Verify that scrolling over the Tags List causes the top separator to fade in

### Test: Notes List
1. Log into the app
2. Click over any row in your Notes List

- [ ] Verify the selected Note is highlighted with a (full width) background color

### Test: Metrics
1. Log into the app
2. Select any Note
3. Click over the top right `( i )` icon

- [ ] Verify the Note Metrics look great

### Test: Interilnks
1. Log into the app
2. Type `[keyword` (with `keyword` being a word present in any of your note's titles)

- [ ] Verify the Interlinking UI looks great!

### Release
`RELEASE-NOTES.txt` was updated in b09eaee with:

> Adopting BigSur Inset Style

### Screenshots:

Main UI: Catalina|Main UI: Big Sur
-|-
<img width="300" alt="Big Sur Main" src="https://user-images.githubusercontent.com/1195260/100018437-3c602400-2dbb-11eb-8edb-69bfe3a8385a.png">|<img width="300" alt="Catalina Main" src="https://user-images.githubusercontent.com/1195260/100018444-3f5b1480-2dbb-11eb-998e-334e39b88c99.png">


Metrics UI: Catalina|Metrics UI: Big Sur
-|-
<img width="300" alt="Catalina Metrics" src="https://user-images.githubusercontent.com/1195260/100018554-64e81e00-2dbb-11eb-97ab-134aa11d1322.png">|<img width="300" alt="Big Sur Metrics" src="https://user-images.githubusercontent.com/1195260/100018578-6d405900-2dbb-11eb-8ba3-e54453a28b9d.png">

Interlinks UI: Catalina|Interlinks UI: Big Sur
-|-
<img width="300" alt="Catalina Interlinks" src="https://user-images.githubusercontent.com/1195260/100018626-8517dd00-2dbb-11eb-8b87-87ca11bf4e4c.png">|<img width="300" alt="Big Sur Interlinks" src="https://user-images.githubusercontent.com/1195260/100018630-86e1a080-2dbb-11eb-9f41-a22a69f5b9eb.png">
